regulations/static/regulations/css/less/layout.less

```diff
@@ -4,6 +4,11 @@
  layout.less defines the structural layout of the site
  */
 
+// focus for accessibliity tabbing
+a:focus {
+    border: 1px dotted @black;
+}
```

Comment:
I discovered that outlines were being set to `0` in the TOC, so I removed those and added the `outline` focus in normalize.

I don't think it's visible enough for the TOC tabs but it's better than before:
<img width="239" alt="screen shot 2016-06-07 at 11 02 21 am" src="https://cloud.githubusercontent.com/assets/24054/15869065/8e935aac-2c9f-11e6-9fd2-22fb32d1e139.png">